.gitignore

```diff
@@ -1,6 +1,6 @@
 # Explicitly not ignoring node_modules so that they are included in package downloaded by runner
```

Comment:
the comment is no longer true for not ignoring node_modules

Reply:
I'll fix that.  Thanks!